rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -308,10 +334,16 @@ object RSpace {
       spanF: Span[F],
       par: Parallel[F]
   ): F[ISpace[F, C, P, A, K]] =
-    setUp[F, C, P, A, K](dataDir, mapSize, branch).map {
-      case (historyReader, store) =>
-        new RSpace[F, C, P, A, K](historyReader, AtomicAny(store), branch)
-    }
+    for {
+      setup                  <- setUp[F, C, P, A, K](dataDir, mapSize, branch)
+      (historyReader, store) = setup
+      hotStore               <- HotStore.empty(historyReader)(sk.toCodec, concurrent)
```

Comment:
Can we just use `store`, created and returned from setup? The same in _createReplay_ function.